custom_components/better_thermostat/models/devices/SPZB0001.yaml

```diff
@@ -2,4 +2,5 @@ calibration_type: 0
 calibration_round: false
 has_system_mode: True
 mode_map:
-  heat: auto
\ No newline at end of file
+  heat: auto
+  auto: heat
```

Comment:
Why you add this here?

We have the reverse_modes function and use this in den inbound check.
In this yaml file, the left side represent the mode on the TRV and the right the mode on HA

So this is a inconsistency

Reply:
Well, I was testing, it wasn't working. I rewrote it simpler (to be able to map just the single case we're seeing).